src/emissions-workspace/emissions.service.ts

```diff
@@ -28,25 +30,33 @@ export class EmissionsWorkspaceService {
   async export(params: EmissionsParamsDTO): Promise<EmissionsDTO> {
     const promises = [];
     const DAILY_TEST_SUMMARIES = 0;
+    const MATS_DERIVED_HOURLY_VALUES = 1;
```

Comment:
MATS_DERIVED_HOURLY_VALUES variable shouldn't exist and instead the hourly operating variable should be used and in the hourly operating service mats derived hourly values should be assigned there